### DISCUSSION TITLE
engines 'py,python' not available on host <- no valid engine found

### DISCUSSION THREAD

**Discussion by Cry-bo** [projectdiscovery/nuclei#4491] [Q&A]
大佬们这个怎么解决啊

**Comment by ehsandeep** [comment]
as the error says, it means you don't have python installed on your system, make sure you have program installed on your system that you are looking to use under engine.

**Reply by qianbenhyu** [reply]
but when my python.exe was renamed as "python3.exe"(my pc has python2 env and python3 env), it seems not to work, how can i do with it.
